src/api.js

```diff
@@ -0,0 +1,86 @@
+let debug = require('debug')('app:main');
+let APIBuilder = require('taskcluster-lib-api');
+let taskcluster = require('taskcluster-client');
+let uuid = require('uuid');
+
+let builder = new APIBuilder({
+  title:    'AMQP Messages API Documentation',
+  description: [
+    'This service is responsible for making pulse messages accessible',
+    'from browsers and cli. There are API endpoints to',
+    'bind / unbind to an exchange and pause / resume listening from a queue',
+  ].join('\n'),
+  serviceName: 'tcevents',
```

Comment:
probably just `events`, but I'm not sure... check what other services do, maybe ask @imbstack  :)